storage/mysql/storage_test.go

```diff
@@ -166,9 +157,83 @@ func nodesAreEqual(lhs []storage.Node, rhs []storage.Node) error {
 	return nil
 }
 
+func openTestDBOrDie() *sql.DB {
+	db, err := OpenDB("test:zaphod@tcp(127.0.0.1:3306)/test")
+	if err != nil {
+		panic(err)
+	}
+	return db
+}
+
+// cleanTestDB deletes all the entries in the database. Only use this with a test database
+func cleanTestDB(db *sql.DB) {
```

Comment:
Consider using a variant of the test specific new db logic in `testonly/integration` which creates a separate database for each test.